src/Controls/src/Core/Shapes/Shape.cs

```diff
@@ -440,6 +440,12 @@ protected override Size MeasureOverride(double widthConstraint, double heightCon
 			result.Height += StrokeThickness;
 			result.Width += StrokeThickness;
 
+			if (this is Line or Path or Polyline)
```

Comment:
Why is required for Line, PolyLine and Path and not for example for Polyline?

Reply:
@jsuarezruiz, In non-closed shapes such as Line, Polyline, and Path, the margin is not considered when the shape is inside a StackLayout and no explicit size is defined. The base size value is returned before measuring the path size because the measured result includes the margin, causing the shape to be invisible.
When the height or width is not explicitly set for these shapes, the shape size is reset using boundsByFlattening and does not account for the margin. Hence, the margin is now added at the end of non-closed shapes